hbase-server/src/main/java/org/apache/hadoop/hbase/quotas/RegionServerSpaceQuotaManager.java

```diff
@@ -98,11 +98,11 @@ public synchronized void start() throws IOException {
 
   public synchronized void stop() {
     if (spaceQuotaRefresher != null) {
-      spaceQuotaRefresher.cancel();
```

Comment:
`cancel()` is nothing but `cancel(true)`, so all `cancel()` should be replaced with `shutdown(true)` just like all `cancel(true)` are replaced with `shutdown(true)`

Reply:
shutdown() is just shutdown(true) after fixing the above typo.